apps/opik-backend/config.yml

```diff
@@ -68,6 +68,10 @@ server:
   gzip:
     enabled: true
 
+batchOperations:
+  datasets:
+    maxExperimentInClauseSize: ${BATCH_OPERATIONS_MAX_EXPERIMENT_IN_CLAUSE_SIZE:-5000}
```

Comment:
Minor: this is a quite internal config that probably won't change across installations. Most likely, it could just be a constant in the code. However, for testing purposes you're defining a smaller value, which makes sense. Additionally, the env var variable might not be needed at all. Anyway, it doesn't harm, so feel free to not to change anything here.